translators/postgres_test.go

```diff
@@ -11,7 +11,8 @@ var pgt = translators.NewPostgres()
 func (p *PostgreSQLSuite) Test_Postgres_CreateTable() {
 	r := p.Require()
 	ddl := `CREATE TABLE "users" (
-"id" SERIAL PRIMARY KEY,
+PRIMARY KEY("id"),
```

Comment:
As mentioned previously, the MySQL driver have another output. We should have the same order (column def, then PRIMARY constraint) as in the MySQL driver.